pages/en/committee.markdown

```diff
@@ -52,7 +52,9 @@ David is a bioinformatics student with a keen interest in both genomic data prod
 
 Nanjala is a junior bioinformatician at ICIPE. She currently leads the bionformatics mentorship and incubation program, aimed at increasing the number of well trained bioinformaticians to support genomics research. She is interested in understanding the genetics of HLA and its association to complex immune mediated traits. She is also passionate about capacity building and she has been involved in different projects that support capacity building. Nanjala has formal training in Bioinformatics and Microbiology.
   
-## Sarah Nyanchera
-![Add Profile Photo](https://sbcf.fr/wp-content/uploads/2018/03/sbcf-default-avatar.png)
+## Sarah Nyanchera Nyakeri
+![Sarah_Nyakeri_N](https://user-images.githubusercontent.com/45894410/162281165-9eb9ad82-7e24-48ee-9c00-d5b0c76c1f93.jpg)
```

Comment:
```suggestion
![Sarah_Nyakeri_N](https://user-images.githubusercontent.com/45894410/162281165-9eb9ad82-7e24-48ee-9c00-d5b0c76c1f93.jpg){: height="150" width="150" }
```

Reply:
@newbiochemist Can you apply this change, so we can merge?

Reply:
@bkmgit apply and merge, it will be faster

Reply:
@Fnyasimi @newbiochemist It is not in my fork, so cannot apply your change directly.